src/main/java/org/snomed/aag/data/domain/ProjectAcceptanceCriteria.java

```diff
@@ -139,6 +139,29 @@ public void addToSelectedTaskCriteria(CriteriaItem criteriaItem) {
 		}
 	}
 
+	public void setSelectedCriteria(Set<CriteriaItem> criteriaItems) {
+		int projectCount = 0;
+		int taskCount = 0;
+		for (CriteriaItem criteriaItem : criteriaItems) {
```

Comment:
This will take a collection of CriteriaItems and either add their identifiers to the project or task collection.